app/domain/export/ics/events.rb

```diff
@@ -10,22 +10,34 @@ class Events
 
     def generate(events)
       ical = Icalendar::Calendar.new
-      ical_events = events.map { |event| generate_ical_events(event) }.flatten
+      ical_events = events.map { |event| generate_ical_from_event_dates(event) }.flatten
       ical_events.each { |event| ical.add_event(event) }
       ical.to_ical
     end
 
-    def generate_ical_events(event)
+    def generate_ical_from_event_dates(event)
       event.dates.map do |event_date|
-        Icalendar::Event.new.tap do |ical_event|
-          ical_event.dtstart = event_date.start_at
-          ical_event.dtend   = event_date.finish_at
-          ical_event.summary = "#{event.name}: #{event_date.label}"
-          ical_event.location = event_date.location || event.location
-          ical_event.description = event.description
-          ical_event.contact = event.contact && event.contact.to_s
-        end
+        generate_ical_from_event_date(event_date, event)
       end
     end
+
+    def generate_ical_from_event_date(event_date, event)
+      Icalendar::Event.new.tap do |ical_event|
+        ical_event.dtstart = datetime_to_ical(event_date.start_at)
+        ical_event.dtend   = datetime_to_ical(event_date.finish_at)
+        ical_event.summary = "#{event.name}: #{event_date.label}"
+        ical_event.location = event_date.location || event.location
+        ical_event.description = event.description
+        ical_event.contact = event.contact && event.contact.to_s
```

Comment:
Maybe I am narrow-minded, but the alignment of the `=` bothers me a little. since this is as non-functional as it gets, I would only change it if you feel the same way.

Reply:
Alignment was not intentional, getting rid of it.